ush/config_defaults.sh

```diff
@@ -501,6 +503,7 @@ FCST_LEN_HRS_SPINUP="1"
 FCST_LEN_HRS_CYCLES=( )
 DA_CYCLE_INTERV="3"
 RESTART_INTERVAL="3,6"
+REGIONAL_ENSEMBLE_OPTION="1"
```

Comment:
This is duplicate of line 594:
regional_ensemble_option=1     #1 for GDAS ; 5 for FV3LAM ensemble